### ISSUE TITLE
warning with building the vignette, `job_logging.Rmd`

### ISSUE THREAD

**Issue by pbchase** [ctsit/redcapcustodian#141]
@ChemiKyle reported a warning with building the vignette, `job_logging.Rmd` in issue #112. I have moved the complaint to its own issue as I am closing #112

`devtools::check()`

```R
  --- re-building ‘job_logging.Rmd’ using rmarkdown
  Quitting from lines 18-22 (job_logging.Rmd) 
  Error: processing vignette 'job_logging.Rmd' failed with diagnostics:
  dot-env file does not exist
  --- failed re-building ‘job_logging.Rmd’
```

```R
N  checking for unstated dependencies in vignettes
   '::' or ':::' imports not declared from:
     ‘dotenv’ ‘here’
   'library' or 'require' calls not declared from:
     ‘kableExtra’ ‘tidyverse’
```

**Comment by pbchase** [comment]
I rectified the dotenv, here, kableExtra and tidyverse errors, but then got this:

```
-- RMarkdown error -------------------------------------------------------------

Quitting from lines 105-113 [iris-updates] (job_logging.Rmd)
Warning messages:
1: Connection is garbage-collected, use dbDisconnect() to avoid this. 
2: Database is garbage-collected, use dbDisconnect(con, shutdown=TRUE) or duckdb::duckdb_shutdown(drv) to avoid this. 
--------------------------------------------------------------------------------
Error in `map()`:
ℹ In index: 8.
Caused by error in `render_rmarkdown()`:
! Failed to render RMarkdown
Caused by error:
! in callr subprocess.
Caused by error:
! unable to find an inherited method for function 'dbGetInfo' for signature '"logical"'
Backtrace:
    ▆
 1. └─redcapcustodian::log_job_success(jsonlite::toJSON(log_message))
 2.   └─redcapcustodian:::write_success_job_log_entry(...)
 3.     └─DBI::dbGetInfo(con)
 4.       └─methods (local) `<fn>`(`<list>`, `<stndrdGn>`, `<env>`)
```

**Comment by pbchase** [comment]
Addressed by PR #142
